src/components/tabs/Tabs.styles.ts

```diff
@@ -0,0 +1,50 @@
+import styled from 'styled-components';
+
+export const TabsContainer = styled.div`
+  display: flex;
+  width: 100%;
+  gap: 0.5rem;
+  overflow-x: auto;
+
+  /* 스크롤바 안 보이게 설정 */
+  -webkit-overflow-scrolling: touch;
+  scrollbar-width: none;
+  -ms-overflow-style: none;
+
+  &::-webkit-scrollbar {
+    display: none;
+  }
+`;
+
+export const Tab = styled.button<{ isActive: boolean }>`
+  display: flex;
+  padding: 0.25rem 0.5rem;
+  justify-content: center;
+  align-items: center;
+  gap: 0.625rem;
+  border-radius: 1.25rem;
+  transition:
+    background-color 0.3s ease,
+    border 0.3s ease;
+  cursor: pointer;
+  border: ${(props) =>
+    props.isActive ? 'none' : `0.6px solid ${props.theme.colors.grayScale.white}`};
+  background-color: ${(props) =>
+    props.isActive ? props.theme.colors.primary.bl400 : props.theme.colors.grayScale.black};
```

Comment:
배경색이 black이 아니라 투명이 되어야 할 것 같아요. 지도쪽에서 투명한 배경이 필요합니다.
```suggestion
    props.isActive ? props.theme.colors.primary.bl400 : 'transparent'};
```